wpengine/faustjs
# fix: prevent 500 error on archives

Archives such as `/category/uncategorized` will no longer produce a 500 error. For [BH-864](https://wpengine.atlassian.net/browse/BH-864).

### To test
From the project root:
- `npm run clean` (if not using a clean clone)
- `npm run bootstrap`
- `npm run dev`

Visit http://localhost:3000/category/uncategorized. You should see a list of posts instead of a 500 error.

### Notes
- Posts are currently latest posts instead of category-specific posts. This will be fixed in a future PR and depends on ongoing work in the custom-queries branch.
- Will need merge conflicts resolving once https://github.com/wpengine/headless-framework/pull/122 is merged.
